### PR TITLE
Update ULTIMATE

### PR DESCRIPTION
**Pull request by jkramer7**
unceil isn't a real function.

### CODE REVIEW

File: ULTIMATE [hunk 1]
```diff
@@ -14,22 +14,23 @@ Games
...
+    - roll MegaDie M = abs( outer - inner )
+    - make ceil(M!/2^(M-1)) valid moves and win points = M. You must calculate this by hand.
+    - if it falls, lose points = 15 - M
 + Scrabble
     - roll 8 sided (R) and pick that number of tiles
     - make a valid play and win points = 9 - R
-    - fail to play and lose points = R
+    - fail to play and lose points = 2R
 + Smart Ass
-    - roll 8 sided (Ra) and have turns = unceil( 2Ra )
```
Comment (areinisc) [inline ULTIMATE:25]: This is a perfect example of unceil in its natural glory.

**Comment by areinisc** [comment]
These changes? Are they necessary?

**Comment by jkramer7** [comment]
image

**Comment by areinisc** [comment]
ahhh... a compelling argument my friend

**Pull request merged**